facebook-commerce-pixel-event.php

```diff
@@ -575,8 +571,7 @@ public static function get_use_pii_key() {
 			if ( ! $fb_options ) {
 				return null;
```

Comment:
Afaik `return` will evaluate to `NULL`, and I prefer functions to always return something, especially because everywhere in the code we use this function on the right side of the equality.